src/Modules/ModParser.lua

```diff
@@ -48,6 +48,7 @@ local formList = {
 	["^([%+%-]?[%d%.]+)%%? of"] = "BASE",
 	["^([%+%-][%d%.]+)%%? base"] = "BASE",
 	["^([%+%-]?[%d%.]+)%%? additional"] = "BASE",
+	["cost ([%+%-]%d+) mana"] = "BASEMANACOST",
```

Comment:
This could probably be generalised to other costs e.g. energy shield, mana, life and rage.

Reply:
Generalized as suggested.